spring-integration-file/src/main/java/org/springframework/integration/file/config/AbstractRemoteFileInboundChannelAdapterParser.java

```diff
@@ -57,7 +57,8 @@ protected final BeanMetadataElement parseSource(Element element, ParserContext p
 		String remoteFileSeparator = element.getAttribute("remote-file-separator");
 		synchronizerBuilder.addPropertyValue("remoteFileSeparator", remoteFileSeparator);
 		IntegrationNamespaceUtils.setValueIfAttributeDefined(synchronizerBuilder, element, "temporary-file-suffix");
-		this.configureFilter(synchronizerBuilder, element, parserContext);
+		FileParserUtils.configureFilter(synchronizerBuilder, element, parserContext,
+				getSimplePatternFileListFilterClassname(), getRegexPatternFileListFilterClassname());
```

Comment:
If we are refactoring this already, maybe let's switch to `Class<?>`.
My hands itch to fix the `...Classname()` camel-case typo, when I see this code :smile: